src/engine/ruleset/decoders/sophos/utm/sophos-utm.yml

```diff
@@ -0,0 +1,188 @@
+
+    # 2017:3:18-18:24:33 hostname reverseproxy: [tutla] [isund:high] [pid 3136:uidex] [client uptate] Invalid signature, cookie: JSESSIONID
+    # 2016:3:26-10:20:16 ctetura3009.www5.corp reverseproxy: [lita] [adeseru:medium] [pid 7692:eaq] amest configured -- corp normal operations
+    - event.original: <@timestamp/%Y:%m:%d-%H:%M:%S> <host.hostname> <event.code>:<~/ignore/ >[<~>] [<~tmp.eventlog>:<~tmp.severity>] [pid <~tmp.process_pid>:<~>] <~tmp.message>
+
+    # 2016:2:12-13:12:33 astarosg_TVM[5716]: id=ommod severity=medium sys=inima sub=tlabo name=web request blocked, forbidden application detectedaction=accept method=ugiatnu client=stiae facility=nofdeF user=sunt srcip=10.57.170.140 dstip=10.213.231.72 version=1.5102 storage=emips ad_domain=imadmi object=ostrume class=molest type=upt attributes=uiineavocount=tisetq node=irati account=icistatuscode=giatquov cached=eritquii profile=dexeac filteraction=iscinge size=6992 request=oreseos url=https://mail.example.net/tati/utaliqu.html?iquaUten=santium#iciatisu referer=https://www5.example.org/eporroqu/uat.txt?atquovo=suntinc#xeac error=nidolo authtime=tatn dnstime=eli cattime=nnu avscantime=dolo fullreqtime=Loremip device=idolor auth=emeumfu ua=CSed exceptions=lupt group=psaquae category=oinBCSe categoryname=mnisist content-type=sedd reputation=uatD application=iunt app-id=temveleu reason=colabo filename=eme file=numqu extension=qui time=civeli function=block line=agnaali message=gnam fwrule=tat seq=ipitla initf=enp0s7281 outitf=enp0s7084 dstmac=01:00:5e:de:94:f6 srcmac=01:00:5e:1d:c1:c0 proto=den length=tutla tos=olorema prec=;iades ttl=siarchi srcport=2289 dstport=3920 tcpflags=mqu info=apariat prec=tlabore caller=untmolli engine=remi localip=saute host=ercit2385.internal.home extra=run server=10.47.202.102 cookie=quirat set-cookie=llu
+    - event.original: <@timestamp/%Y:%m:%d-%H:%M:%S> <event.code>[<~tmp.process_pid>]:<~/ignore/ ><~tmp.message>
+
+    # 2016:3:12-03:17:42 data4478.api.lan confd: id=iquipex severity=very-high sys=uradip sub=wri name=bor client=occa facility=stquidol user=itquiin srcip=10.106.239.55 version=1.3129 storage=atevel object=nsecte class=itame type=eumfug attributes=litcount=asun node=estia account=eaq
+    - event.original: <@timestamp/%Y:%m:%d-%H:%M:%S> <host.hostname> <event.code>:<~/ignore/ ><~tmp.message>
+
+# TODO: All sophos named fields are not currently in the schema.
+normalize:
+
+  - check: event.code==confd OR event.code==aua
+    logpar:
+      - ~tmp.message: <~tmp/kv/=/ /'/'>
+    map:
+      - file.directory: $~tmp.storage
+      - log.level: $~tmp.severity
+      - observer.version: $~tmp.version
+      - related.hosts: $~tmp.host.hostname
+      - related.ip: $~tmp.srcip
+      - rsa.identity.logon_type: $~tmp.facility
+      - rsa.internal.event_desc: $~tmp.name
+      - rsa.internal.messageid: $~tmp.event.code
```

Comment:
Is this field always mapped as event.code? Then you could map it at the end, one time instead of mapping it on every check-map